velox/functions/lib/TimeUtils.h

```diff
@@ -31,6 +31,12 @@ inline constexpr int64_t kSecondsInDay = 86'400;
 inline constexpr int64_t kDaysInWeek = 7;
 extern const folly::F14FastMap<std::string, int8_t> kDayOfWeekNames;
 
+FOLLY_ALWAYS_INLINE bool isTimeUnit(const DateTimeUnit unit) {
```

Comment:
It seems this functions corresponds to the `DateTimeUnit` in 'DateTimeFormatter.h', because nanosecond is also time unit but not considered. Perhaps move it near the `DateTimeUnit`.